Sources/KVKCalendar/Style.swift

```diff
@@ -102,6 +102,7 @@ public struct TimelineStyle {
     public var eventFont: UIFont = .boldSystemFont(ofSize: 12)
     public var offsetEvent: CGFloat = 3
     public var startHour: Int = 0
+    public var scrollToHour: Int = 0
```

Comment:
Could you mark the property as optional, pls?

Reply:
Good. I made a new commit